bench/ArrayBench.java

```diff
@@ -0,0 +1,30 @@
+package benchmark;
+
+import com.sun.cldchi.jvm.JVM;
+
+public class ArrayBench {
+    void runBenchmark() {
+        long start, time;
+        int len = 1;
+        int iterations = 1000;
+
+        System.out.println("Time to allocate int arrays of various lengths " + iterations + " times, by length, in ms:");
+
+        for (int i = 0; i < 20; i++) {
+
+            start = JVM.monotonicTimeMillis();
+            for (int j = 0; j < iterations; j++) {
```

Comment:
> To see differences when the length of the array is small, we could use a larger number of iterations.

Both kinds of benchmarks would be useful, so I've added a variable iteration benchmark in c42eeea. Results look like this:

> Time to allocate int arrays of variable lengths a constant 1000 times, by length, in ms: 
> ArrayBench-1: 6 
> ArrayBench-2: 2 
> ArrayBench-4: 2 
> ArrayBench-8: 2 
> ArrayBench-16: 1 
> ArrayBench-32: 2 
> ArrayBench-64: 2 
> ArrayBench-128: 2 
> ArrayBench-256: 3 
> ArrayBench-512: 7 
> ArrayBench-1024: 7 
> ArrayBench-2048: 31 
> ArrayBench-4096: 37 
> ArrayBench-8192: 25 
> ArrayBench-16384: 39 
> ArrayBench-32768: 34 
> ArrayBench-65536: 56 
> ArrayBench-131072: 92 
> ArrayBench-262144: 163 
> ArrayBench-524288: 327 
> Time to allocate int arrays of variable lengths a variable number of times, by length * iterations, in ms: 
> ArrayBench-1*524288: 225 
> ArrayBench-2*262144: 74 
> ArrayBench-4*131072: 37 
> ArrayBench-8*65536: 21 
> ArrayBench-16*32768: 11 
> ArrayBench-32*16384: 5 
> ArrayBench-64*8192: 3 
> ArrayBench-128*4096: 2 
> ArrayBench-256*2048: 1 
> ArrayBench-512*1024: 4 
> ArrayBench-1024*512: 2 
> ArrayBench-2048*256: 1 
> ArrayBench-4096*128: 1 
> ArrayBench-8192*64: 1 
> ArrayBench-16384*32: 1 
> ArrayBench-32768*16: 1 
> ArrayBench-65536*8: 1 
> ArrayBench-131072*4: 1 
> ArrayBench-262144*2: 0 
> ArrayBench-524288*1: 0